src/chat.rs

```diff
@@ -921,12 +921,13 @@ fn get_draft_msg_id(context: &Context, chat_id: u32) -> u32 {
         .unwrap_or_default() as u32
 }
 
-pub unsafe fn get_draft(context: &Context, chat_id: u32) -> Result<Message, Error> {
+pub fn get_draft(context: &Context, chat_id: u32) -> Result<Option<Message>, Error> {
     ensure!(chat_id > DC_CHAT_ID_LAST_SPECIAL, "Invalid chat ID");
     let draft_msg_id = get_draft_msg_id(context, chat_id);
-    ensure!(draft_msg_id != 0, "Invalid draft message ID");
-
-    dc_msg_load_from_db(context, draft_msg_id)
+    if draft_msg_id == 0 {
+        return Ok(None);
+    }
+    Ok(Some(dc_msg_load_from_db(context, draft_msg_id).unwrap()))
```

Comment:
uhm why is there an `unwrap` call in here? 

this should probably be sth like

```rust
let draft = dc_msg_load_from_db(context, draft_msg_id)?;
Ok(Some(draft))
```

Reply:
done